app/views/resources/_collaborators.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="collaborators_avatar">
```

Comment:
Render collaborators associated with a group and collaborators NOT associated with a group in separate places of the view.